CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+<!----->
+<!--- Copyright (c) 2018-2024 ETH Zurich -->
+<!----->
+<!--- SPDX-License-Identifier: BSL-1.0 -->
+<!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
+<!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
```

Comment:
License should be BSD3, shouldn't it?

Reply:
Yep, good catch; I've just copy-pasted from pika. No license? I'm honestly not sure what we should put here. I've just removed the header since we don't have it in other `.md` files, but if this one needs a license then the others should have one as well.

Reply:
> No license

I don't think it's needed/useful here, but others might have a different opinion.